### PR TITLE
Fix error parsing of device_is_ab

### PR DESCRIPTION
**Pull request by tiann**
Please be aware that:

```python
In [1]: bool("false")
Out[1]: True
```

### CODE REVIEW

File: twrpdtgen/info_extractors/buildprop.py [hunk 1]
```diff
@@ -38,7 +39,7 @@ def __init__(self, build_prop: BuildProp):
 			self.platform = "default"
 
 		try:
-			self.device_is_ab = bool(self.get_prop(DEVICE_IS_AB, "A/B"))
+			self.device_is_ab = bool(strtobool(self.get_prop(DEVICE_IS_AB, "A/B")))
```
Comment (SebaUbuntu) [inline twrpdtgen/info_extractors/buildprop.py:42]: Is the cast to bool needed after strtobool function?

Comment (tiann) [reply twrpdtgen/info_extractors/buildprop.py:42]: > Is the cast to bool needed after strtobool function?

Yes, `strtobool` return int

Comment (SebaUbuntu) [reply twrpdtgen/info_extractors/buildprop.py:42]: Alright then, `strtobool` is kinda a misleading function name, `strtoint` would have been better XD

**Comment by SebaUbuntu** [comment]
Thank you 👍

**Pull request merged**